### PR TITLE
Feature/starter code

### PR DESCRIPTION
**Pull request by martinmags**
- Added axios, material ui
- TODO: setup folders
- cleaned up

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,46 @@
...
+You will also see any lint errors in the console.
 
-_Pending_
+### `npm test`
 
-# Technology
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-- Frontend Framework: React
```
Comment (nicolegooden) [inline README.md:19]: I feel like it could be helpful to keep this section in here (from line 19 to line 26)!

File: README.md [hunk 2]
```diff
@@ -1,33 +1,46 @@
...
-- UI Framework: Material-UI
-- Backend Web Framework: Express
-- Language: TypeScript
-- Database: Mongodb
-- DevOps: Git workflow, Branch Protection
-- Project Management: Slack, Figma, Trello, GitHub Wiki
-- Host: Heroku
+### `npm run build`
 
-# Authors
```
Comment (nicolegooden) [inline README.md:28]: I'd also suggest keeping the Authors section.

File: README.md [hunk 3]
```diff
@@ -1,33 +1,46 @@
-# july-challenge-tbd
+# Getting Started with Create React App
 
-A web app that acts as a directory to find employees based on their areas of expertise and team involvement.
```
Comment (nicolegooden) [inline README.md:3]: I'd also keep this line included so we have a brief description to expand upon later!

File: README.old.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# july-challenge-tbd
```
Comment (nicolegooden) [inline README.old.md:1]: Oh I'm seeing now that the old README content is here instead, oops! Is there a reason to have two separate ones?

**Review by nicolegooden — CHANGES_REQUESTED** [review]
Just left some comments on the README side of things. Thanks for updating this repo so that it contains all of the create-react-app boilerplate!

File: README.old.md (hunk 4 above)
Comment (martinmags) [reply README.old.md:1]: I was so confused for a sec as I was reading through your comments LOL  I thought I pushed the wrong code for a sec. The different READMEs are from my old one and the new one from create-react-app. I can combine them, thanks for the notice!

**Review by nicolegooden — APPROVED** [review]

**Pull request merged**